.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: node_js
+
+node_js:
+  - "10"
+
+sudo: false
+env:
+  - TEST_SUITE=lint
+  - TEST_SUITE=mocha
+
+cache:
+  directories:
+    - node_modules
+
+install:
+  - npm ci
+
+script:
+  - if [ "$TEST_SUITE" == "lint" ]; then npm run lint; else npm run test; fi
```

Comment:
I think we can use one line per test suite and no else so it's easier to read

Reply:
Updated - is this what you were thinking?

Reply:
I was thinking something like this instead:

```bash
if [ "$TEST_SUITE" = "lint" ]; then npm run lint; fi
if [ "$TEST_SUITE" = "mocha" ]; then npm run test; fi
```